### ISSUE TITLE
Icons missing in 0.50.2

### ISSUE THREAD

**Issue by beepsoft** [describo/crate-builder-component#82]
I just checked the latest 0.50.2 version and it seems that icons are not displayed in buttons:

<img width="975" alt="image" src="https://github.com/describo/crate-builder-component/assets/1926265/6630c71a-f599-467f-8f8d-7aedcaa9d396">

**Comment by marcolarosa** [comment]
Ahh - sorry!

I was having problems with element-plus initialisation so I ended up removing that and fontawesome setup from the plugin (https://github.com/describo/crate-builder-component/commit/eaba93608a8eda6ab4e754f84f78091643206c15#diff-03797d0496d3cd9aa299f2dd7379185057c1d82d5efe28df40e27a2941f62e38). I added the element-plus css to the app but not the fontawesome imports (fixed by https://github.com/describo/crate-builder-component/commit/13664a8a9da881ad0015c3697efdcd709cd89099#diff-c40060953cc2aa01da23fa42fbbf3d196850091f87289a1f6ac8d71008342b65).

That said, I'm not sure how this will affect you in the react app. If we need to add this back in to the plugin initialisation code let me know.
